app/controllers/application_controller.rb

```diff
@@ -45,8 +45,30 @@ def failbot_context
     }
   end
 
+  def authorize_google_classroom
```

Comment:
with `app/controllers/application_controller/google_authentication_dependency.rb` not working, I just added the methods to the actual controller. This seemed to fix the issue I was having with the file.